src/sidebar/store/modules/selection.js

```diff
@@ -163,27 +173,38 @@ const setTab = (newTab, oldTab) => {
   };
 };
 
+/**
+ * Return state object representing a reset selection. This
+ * resets user-set filters (but leaves focus mode intact)
+ *
+ * @return {Object}
+ */
+const resetSelection = () => {
```

Comment:
Given that store state is immutable, this can just be a constant. You could wrap the literal with the `immutable` helper if you wanted to guard against accidental mutation.

Reply:
I have a little bit of refactoring to do in this module as a follow up (e.g. `clearSelection` and `clearSelectedAnnotations` actions now do the same, so one should be eliminated). I'll put this on the list—cool?